molecule/testinfra/app/test_appenv.py

```diff
@@ -105,16 +105,9 @@ def test_ensure_logo(host):
         assert f.group == "root"
 
 
-def test_securedrop_tmp_clean_cron(host):
-    """Ensure securedrop tmp clean cron job in place"""
-    with host.sudo():
-        cronlist = host.run("crontab -u www-data -l").stdout
-        cronjob = f"@daily {sdvars.securedrop_code}/manage.py clean-tmp"
-        assert cronjob in cronlist
```

Comment:
Would it be worthwhile to add tests for the expected systemd units, especially after the additional changes in `securedrop/debian/rules`?

Reply:
I struggle with balancing what things the package does that we can just trust that it does vs those we should assert. And I don't want to end up in the state where we have two lists of all the units we install and then test..

What would you expect/want a test to check?

Reply:
Very fair.  I can get over my knee-jerk reaction at the asymmetry of—

```
‒ cron jobs
‒ tests for cron jobs
+ systemd timers
```

—given that the latter's declarative trustworthiness is our whole motivation for this switch .  :-)